src/main/flight/mixer.c

```diff
@@ -721,12 +722,37 @@ static void applyFlipOverAfterCrashModeToMotors(void)
     }
 }
 
+static float applyThrustLinearization(float motorOutput)
+{
+#ifdef USE_THRUST_LINEARIZATION
+    if (motorConfig()->thrustLinearization) {
+        float tl = motorConfig()->thrustLinearization * 0.01f;
+        static float a, a_reci, b, b_sq;
+        if (a != tl) {
+            a = tl;
+            a_reci = 1.0f / a;
+            b = (1.0f - a) / (2.0f * a);
+            b_sq = b * b;
+        }
+
+        if (motorOutput > 0.0f) {
+            motorOutput = 1.0f / fast_rsqrt(motorOutput * a_reci + b_sq) - b;
```

Comment:
(b_sq is not necessary, multiplication is 1 cycle, faster than fetch from memory)

Reply:
thank you